cypress/e2e/functional/document_tests/student_teacher_4up_readonly_spec.js

```diff
@@ -131,6 +129,7 @@ function setupTestBrain(studentIndex) {
   simulatorTile.getTileTitle().should("contain", "Simulation 1");
   simulatorTile.getSimulatorTileTitle().click();
   simulatorTile.getSimulatorTileTitle().type(newName + '{enter}');
+  cy.wait(1000);
```

Comment:
Probably not needed but a couple of times I saw only part of the `newName` value rendered in the tile without the wait.